go.sum

```diff
@@ -1,4 +1,5 @@
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/goreflect/go_structor v0.0.0-20200116205716-48742d4c2b6e h1:MRl+84ypcf8lvNTlxPHU7PZxdTfBEqzQODgjJC3p5n0=
```

Comment:
don't need it

Reply:
fixed but this file does not affect assembly